app/src/lib/release-notes.ts

```diff
@@ -0,0 +1,105 @@
+const ordinal: (i: number) => string = require('ordinal')
+
+import {
+  ReleaseMetadata,
+  ReleaseNote,
+  ReleaseSummary,
+} from '../models/release-notes'
+
+const itemEntryRe = /^\[(new|fixed|improved|removed|added|pretext)\]\s(.*)/i
+
+const monthNames = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+]
+
+function formatDate(date: Date) {
```

Comment:
Is the primary motivation for this method of rendering a date to a string the ordinal suffixes? I'm asking because everything but that could be replaced with:

```js
const options = { year: 'numeric', month: 'long', day: 'numeric' };
console.log(date.toLocaleDateString('en-US', options));
```

and the ordinal suffix could be spliced in. This would remove us needing to enumerate our own months list above.

Reply:
I should have dug deeper into this - the release date displayed is supposed to include the day suffix - so `1st`, `2nd`, `3rd` or whatever, but this snippet doesn't do so as-is. I just tweaked the tests to check (they  avoid validating the day itself because timezones) - digging into this now but let me know if you can spot what's missing:

```
    console.log app/test/unit/release-notes-test.ts:118
      date: December 5, 2017
    console.log app/test/unit/release-notes-test.ts:146
      date: November 16, 2017
```

Reply:
b73e73984 switches over to use `moment` which supports the full range of formats